src/applications/claims-status/components/ClaimDetailLayout.jsx

```diff
@@ -23,13 +34,12 @@ export default class ClaimDetailLayout extends React.Component {
 
     let content;
     if (!loading) {
-      const claimsPath = `your-claims${claim.attributes.open ? '' : '/closed'}`;
```

Comment:
Is this change intentional?

Reply:
I've restored the previous logic, but I'm not sure we need it. The `routes.jsx` file only had routing for `/closed` in commented out, old versions of the file. Doesn't hurt to leave it though.